server/shared/termdb.usecase.js

```diff
@@ -179,6 +179,18 @@ export function isUsableTerm(term, _usecase, ds) {
 				return uses
 			}
 
+		case 'filter':
+			if (graphableTypes.has(term.type)) {
+				if (usecase.detail == 'GDC') {
+					// is gdc ds. api doesn't support filtering by survival, do not show survival
+					if (term.type != 'survival') uses.add('plot')
+				} else {
+					uses.add('plot')
+				}
+			}
+			if (!term.isleaf) uses.add('branch')
```

Comment:
Should use `if (hasNonSurvivalTermChild(child_types)) uses.add('branch')` here.? The way it's written above, the survival branch is assumed to never have a non-survival term as a descendant, but it's better to be flexible now instead of potentially having to revisit this logic later.